src/app.js

```diff
@@ -10,12 +10,13 @@ const loadConfig = require('./lib/config')
 const RoutingTables = require('./lib/routing-tables')
 const RouteBuilder = require('./lib/route-builder')
 const RouteBroadcaster = require('./lib/route-broadcaster')
+const CurveCache = require('./lib/curve-cache')
```

Comment:
Whether `expiresAt` is honored is ultimately a per-peering-relationship legal question.  My focus on route-selection heuristics is partly based on the assumption that games will be played with this.